Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -49,6 +57,29 @@ ABenchmarkGymGameMode::ABenchmarkGymGameMode()
 
 	NPCSToSpawn = 0;
 	SecondsTillPlayerCheck = 15.0f * 60.0f;
+
+	bTestEnabled = false;
+	FParse::Bool(FCommandLine::Get(), TEXT("-NFRTestEnabled="), bTestEnabled);
```

Comment:
Can we set these the same way we set the other flags? ie. worker flags or in native, via command line?